services/collaboration/pkg/config/grpc.go

```diff
@@ -0,0 +1,7 @@
+package config
+
+// Service defines the available grpc configuration.
+type GRPC struct {
+	Addr      string `yaml:"addr" env:"COLLABORATION_GRPC_ADDR" desc:"The bind address of the GRPC service"`
```

Comment:
```suggestion
	Addr      string `yaml:"addr" env:"COLLABORATION_GRPC_ADDR" desc:"The bind address of the GRPC service."`
```
Q: does this have anything to do with the global `OCIS_GATEWAY_GRPC_ADDR` envvar located in `gateway` and `storage-users`? If this is the same, you need to write `OCIS_GATEWAY_GRPC_ADDR;COLLABORATION_GRPC_ADDR`

Reply:
"collaboration" starts a GRPC service and a HTTP service. The address refers to this particular GRPC service.
It doesn't refer to the ocis' gateway